maven-core/src/main/java/org/apache/maven/toolchain/DefaultToolchainManager.java

```diff
@@ -43,14 +46,24 @@
 public class DefaultToolchainManager
     implements ToolchainManager
 {
-    final Logger logger;
+    protected final Logger logger; // TODO this class is extended, needs refactoring
```

Comment:
These two classes DefaultToolchainManager and DefaultToolchainManagerPrivate needs refactor, both has TODOs for it

Reply:
Will handle in https://issues.apache.org/jira/browse/MNG-7280